src/BloomBrowserUI/bookEdit/toolbox/readers/readerToolsModel.ts

```diff
@@ -815,6 +815,12 @@ export class ReaderToolsModel {
     }
 
     var pageStrings = _.values(this.pageIDToText);
+    // In obtaining pageStrings, for some reason lines can be separated by \\n instead of \r\n (or \n?).
```

Comment:
What is the advantage of fixing it downstream, here, vs. back where this weird format appears to start (ReadersApi.cs GetTextOfContentPagesAsJson())?